requirements.txt

```diff
@@ -12,3 +12,4 @@ dockerfile-parse~=0.0
 requests~=2.23
 stevedore>=1.32
 pbr>=5.4
+debut
```

Comment:
Can you give me the version of debut you were using at the point you created the PR and a list of dependencies?

Reply:
@nishakm, here along with `debut` version do I need to add its dependencies also in `requirements.txt`?

Reply:
Nope. We just need to figure out how much the dependencies overlap with the packages we are already using. If there is no overlap then we can add `debut>=major.minor` where major.minor is the version you have. If there is overlap, we will have to add it as `debut~=major.minor`.

Reply:
@nishakm I have updated the `debut` version in `requirements.txt`. Please take a look once